src/scenes/Scene.js

```diff
@@ -2,13 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-THREE.Scene = function () {
+THREE.Scene = function (fog) {
```

Comment:
I don't specifically want to pass ```fog``` to the constructor, I just wanted to try continue supporting everything the existing FBX converter (i.e. ```SceneLoader```) supports. However, I noticed that ```ObjectLoader``` seems to be passing all JSON options from ```data``` as constructor arguments, even the optional ones (in the case of ```HeimisphereLight```):

https://github.com/mrdoob/three.js/blob/dev/src/loaders/ObjectLoader.js#L506-L573

So I just followed suit.